config/formats.js

```diff
@@ -562,6 +562,36 @@ let Formats = [
 		ruleset: ['Pokemon', 'Standard NEXT', 'Team Preview'],
 		banlist: ['Uber'],
 	},
+	{
+		name: '[Gen 7] Shitmons',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Shitmons</a>`,
+		],
+
+		gameType: 'doubles',
+		teamLength: {
+			validate: [2, 2],
+			battle: 2,
+		},
+		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		banlist: [
+			'Honedge', 'Doublade', 'Aegislash', 'Kitsunoh', 'Shedinja ++ Sturdy', 'Power Construct', 'Schooling', 'Iron Barbs', 'Rough Skin',
```

Comment:
``Shedinja ++ Sturdy`` means having the Pokemon Shedinja and the move Sturdy anywhere on the same team (like Shedinja + Sturdy Onix), `Shedinja + Sturdy` means Shedinja is prevented from having Sturdy.